x-pack/plugins/security_solution/public/flyout/document_details/shared/components/flyout_tour.tsx

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/*
+ * This timeline tour only valid for 8.12 release is not needed for 8.13
+ *
+ * */
+
+import type { FC } from 'react';
+import React, { useCallback, useState, useEffect } from 'react';
+import { EuiButton, EuiButtonEmpty, EuiTourStep } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { tourConfig } from './tour_step_config';
+import type { FlyoutTourStepsProps } from './tour_step_config';
+import { NEW_FEATURES_TOUR_STORAGE_KEYS } from '../../../../../common/constants';
+import { useKibana } from '../../../../common/lib/kibana';
+
+interface TourState {
```

Comment:
let's document each property here, to stay consistent with the rest of the flyout code